doc/stdenv/stdenv.chapter.md

```diff
@@ -1057,7 +1057,55 @@ The `validatePkgConfig` hook validates all pkg-config (`.pc`) files in a package
 
 ### cmake
 
-Overrides the default configure phase to run the CMake command. By default, we use the Make generator of CMake. In addition, dependencies are added automatically to CMAKE_PREFIX_PATH so that packages are correctly detected by CMake. Some additional flags are passed in to give similar behavior to configure-based packages. You can disable this hook’s behavior by setting configurePhase to a custom value, or by setting dontUseCmakeConfigure. cmakeFlags controls flags passed only to CMake. By default, parallel building is enabled as CMake supports parallel building almost everywhere. When Ninja is also in use, CMake will detect that and use the ninja generator.
+Overrides the default configure phase to run the CMake command. By default, we use the Make generator of CMake, however, if ninja is present, then the ninja generator will be used instead. In addition, dependencies are added automatically to CMAKE_PREFIX_PATH so that packages are correctly detected by CMake. By default, the cmake configurePhase can be thought of as:
+
+```sh
+mkdir build
+cd build
+cmake .. $cmakeFlags
+```
+
+#### Variables controlling CMake
+
+##### dontUseCmakeConfigure
```

Comment:
Could you please wrap these in backticks like the meson docs do? And optionally add anchors – something like `{#ssec-cmake-vars-dontUseCmakeConfigure}`.

Reply:
I don't see anything like this in the meson section, which is what I mostly copied, however, the `Specifying dependencies` section does